internal/bridge/crunchybridgecluster/helpers_test.go

```diff
@@ -59,102 +54,41 @@ func init() {
 	}
 }
 
-var kubernetes struct {
-	sync.Mutex
-
-	env   *envtest.Environment
-	count int
-}
-
 // setupKubernetes starts or connects to a Kubernetes API and returns a client
-// that uses it. When starting a local API, the client is a member of the
-// "system:masters" group. It also creates any CRDs present in the
-// "/config/crd/bases" directory. When any of these fail, it calls t.Fatal.
-// It deletes CRDs and stops the local API using t.Cleanup.
-//
-// This function was duplicated from the postgrescluster package.
-// TODO: Pull these duplicated functions out into a separate, shared package.
-//
-//nolint:unparam
-func setupKubernetes(t testing.TB) (*envtest.Environment, client.Client) {
+// that uses it. See [require.Kubernetes] for more details.
+func setupKubernetes(t testing.TB) client.Client {
```

Comment:
Could we pull this function and `setupNamespace` out into a helper package rather than having them defined multiple times in different packages? :thinking:

Reply:
Yep, but not me in this PR.

The namespace dumping wasn't helpful for the tests I was writing, so I like having it be optional.

`setupNamespace` is effectively moved to `require.Namespace`, but I didn't want to edit all those calls.